mvpa2/tests/test_searchlight_hyperalignment.py

```diff
@@ -208,7 +208,7 @@ def test_searchlight_hyperalignment(self):
         for ds in dss[1:]:
             assert_false(np.all(ds_orig.samples == ds.samples))
         # testing checks
-        slhyp = SearchlightHyperalignment(ref_ds=1, exclude_from_model=[1])
+        slhyp = SearchlightHyperalignment(ref_ds=2, exclude_from_model=[1])
```

Comment:
So here is the confusing part: `ref_ds` should be the index after excluding datasets according to the documentation.
There're 3 datasets in this test, and 2 left after excluding one of them. Therefore using `ref_ds=1` wouldn't raise `ValueError` while `ref_ds=2` would (the "boundary" case). Similarly, `ref_ds=0` should always work if there's at least one dataset left for training.